package.json

```diff
@@ -17,7 +17,9 @@
     "@tailwindcss/forms": "^0.5.0",
     "next": "12.1.6",
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-bootstrap": "^2.4.0",
```

Comment:
Can we stick to using Tailwind we already using it. since its a template it has to be as minimalistic as possible